rancher2/schema_cluster_rke_config.go

```diff
@@ -137,6 +137,12 @@ func clusterRKEConfigFieldsV0() map[string]*schema.Schema {
 			Computed:    true,
 			Description: "Optional prefix to customize kubernetes path",
 		},
+		"win_prefix_path": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+			Description: "Optional prefix to customize kubernetes path for windows",
+		},
```

Comment:
This added section is not needed here due to `func clusterRKEConfigFieldsV0()` is referencing old tf schema